test/api/info_test.py

```diff
@@ -0,0 +1,17 @@
+"""Test using sqlfluff to extract elements of queries."""
+
+import sqlfluff
+
+
+def test__api__info_dialects():
+    """Basic linting of dialects."""
+    dialects = sqlfluff.dialects()
+    assert isinstance(dialects, list)
+    assert ("ansi", "ansi", "nothing") in dialects
```

Comment:
nothing? Should we filter this out as it's really only an internal thing and everything inherits from ansi?

Reply:
what do you mean "filter it out"? You mean not include the inheritance of a dialect?

Reply:
I mean just prevent `"nothing"` from being returned by `sqlfluff.dialects()`. Definitely include the inheritance

Reply:
Or we could give it a meaningful name if it has to stay in? It's just a bit confusing atm

Reply:
ah  "nothing" isn't being included as a dialect here. `("ansi", "ansi", "nothing")` is a shorthand for ` DialectTuple(label="ansi", name="ansi", inherits_from="nothing")`. Does that make more sense?

Reply:
It does indeed!